homeassistant/components/tradfri/base_class.py

```diff
@@ -69,7 +107,9 @@ def unique_id(self):
     def _observe_update(self, device):
         """Receive new state data for this device."""
         self._refresh(device)
-        self.async_schedule_update_ha_state()
+
+        # Force refresh to handle a situation where the GUI is not updated
+        self.async_schedule_update_ha_state(force_refresh=True)
```

Comment:
Is it tradfri groups that are helped by this change? No other entity has `async_update` implemented, right?